Sources/SubscriberAttributes/SubscriberAttribute.swift

```diff
@@ -32,6 +32,10 @@ struct SubscriberAttribute {
         self.init(withKey: key, value: value, isSynced: false, setTime: dateProvider.now())
     }
 
+    init(attribute: ReservedSubscriberAttribute, value: String?, dateProvider: DateProvider = DateProvider()) {
+        self.init(withKey: attribute.rawValue, value: value, dateProvider: dateProvider)
+    }
+
```

Comment:
do we need this?

Reply:
It cleaned up the call so we sent the `enum` instead of a random `String`. I can remove it since it's only used in 1 place though.